src/hooks/use-suggestions.ts

```diff
@@ -93,8 +98,18 @@ const mergeWords = (words: string[]) => {
 
     for (let i = 0; i < words.length; ++i) {
         const currentItem = words[i]
-
-        if (currentItem === 'IN' || currentItem === 'NOT-IN') {
+        if (
+            currentItem === operators.$exists ||
+            currentItem === operators.$doesnt_exist_dummy
+        ) {
+            merging = false
+            result.push(currentItem)
+            result.push('dummy')
```

Comment:
Seems like debug leftover ?

Reply:
no, it is needed, since we do not have actual value for these operators but other code assumes there is always a value

Reply:
I see, can you export it to a variable with a meaningful name ? It's important so no one will delete that some day

Reply:
done